pipeline_api/src/main/java/com/splicemachine/pipeline/client/BulkWriteAction.java

```diff
@@ -48,6 +51,11 @@
     private static final Logger LOG=Logger.getLogger(BulkWriteAction.class);
     private static final Logger RETRY_LOG=Logger.getLogger(BulkWriteAction.class.getName()+".retries");
     private static final AtomicLong idGen=new AtomicLong(0L);
+    public static final AtomicLong submitTimeCounter=new AtomicLong(0L);
+    public static long submitTime;
+    public static final Cache<byte[],Long> IGNORE_REFRESH = CacheBuilder.newBuilder().
+            concurrencyLevel(64).
+            expireAfterWrite(60,TimeUnit.SECONDS).initialCapacity(128).build();
```

Comment:
Is the .expireAfterWrite() needed? It doesn't seem to serve any function

Reply:
You can technically leak if you put things in the cache and never expire them.  It is a static variable.  The expire ejects them.